WalletWasabi.Gui/Global.cs

```diff
@@ -751,6 +762,12 @@ public async Task DisposeAsync()
 					Logger.LogInfo($"{nameof(TorManager)} is stopped.");
 				}
 
+				var cache = Cache;
+				if (cache is { })
+				{
+					cache.Dispose();
+				}
```

Comment:
Why don't we do this here `cache?.Dispose();`?

Reply:
The suggestion makes sense but i prefer to have this in the exact same way that all the rest.